src/posts/2020-10-14-wcag2.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Introduction to WCAG 2
```

Comment:
```suggestion
title: Introduction to WCAG 2.0
```

Reply:
Hello Eric, I've addressed the issue mentioned above